Simplified/NYPLBookCoverRegistry.m

```diff
@@ -101,7 +101,8 @@ - (void)coverImageForBook:(NYPLBook *)book handler:(void (^)(UIImage *image))han
         if(image) {
           handler(image);
         } else {
-          [self thumbnailImageForBook:book handler:handler];
+          //[self thumbnailImageForBook:book handler:handler];  // why are we calling this again?
```

Comment:
We can't comment this line out otherwise we lose an important feature. I should have done a better job of documenting this code when I wrote it:

- First load the thumbnail image
  - If it has already been cached it returns the cached image
  - If no image is there, then it generates the TenPrintCover
- Now try and load the detail image ( a bigger sized image )
  - **If that image is not there, fall back to the smaller thumbnail image (or a generated image)

** this is the line that you have changed, which now says "if no large image exists, then create a TenPrintCover generated image, which is not preferable to showing the smaller thumbnail slightly blurry.

For now you can revert this back to its original line.